common/src/main/scala/org/apache/celeborn/common/meta/DeviceInfo.scala

```diff
@@ -92,6 +92,10 @@ class DiskInfo(
     this.storageType = storageType
   }
 
+  def setConfiguredUsableSpace(configuredUsableSpace : Long): Unit = {
```

Comment:
No need to add this method since `configuredUsableSpace` is public var

Reply:
done